src/Clearinghouse.sol

```diff
@@ -74,14 +78,16 @@ contract Clearinghouse is Policy, RolesConsumer, CoolerCallback {
     // --- INITIALIZATION --------------------------------------------
 
     constructor(
+        address ohm_,
```

Comment:
You don't need to pass in OHM address. You can grab the OHM address by calling staking.OHM();

I would move this line below line 91, after initializing staking